release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2832,6 +2825,107 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* When you install the {ols-official} that is a Developer Preview feature only, the checkbox for enabling in-cluster monitoring is now enabled by default. (link:https://issues.redhat.com/browse/OCPBUGS-42380[*OCPBUGS-42380*])
+
+* The installation program now uses a newer version of the `cluster-api-provider-ibmcloud` provider that includes Transit Gateway fixes. (link:https://issues.redhat.com/browse/OCPBUGS-42483[*OCPBUGS-42483*])
+
+* The migrating CNS volumes feature that is a Developer Preview feature only includes the following enhancements:
++
+** The feature now checks for the vCenter version before moving CNS volumes to {vmw-first}. (link:https://issues.redhat.com/browse/OCPBUGS-42006[*OCPBUGS-42006*])
+** The feature keeps migrating CNS volumes even if some volumes do not exist, so that the migration operation does not exit when no volume exists. (link:https://issues.redhat.com/browse/OCPBUGS-42008[*OCPBUGS-42008*])
+
+* The vSphere CSI Driver Operator can now deletes all resources for a vSphere CSI driver that was removed from a cluster. (link:https://issues.redhat.com/browse/OCPBUGS-42007[*OCPBUGS-42007*])
+
+[id="ocp-4-17-1-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, the Single-Root I/O Virtualization (SR-IOV) Operator did not expire the acquired lease during the Operator's shutdown operation. This impacted a new instance of the Operator, because the new instance had to wait for the lease to expire before the new instance was operational. With this release, an update to the Operator shutdown logic ensures that the Operator expires the lease when the Operator is shutting down. (link:https://issues.redhat.com/browse/OCPBUGS-37668[*OCPBUGS-37668*])
+
+* Previously, when you configured the image registry to use an {azure-first} storage account that was located in a resource group other than the cluster's resource group, the Image Registry Operator would become degraded. This occured because of a validation error. With this release, an update to the Operator allows for authentication only by using a storage account key. Validation of other authentication requirements is not requried. (link:https://issues.redhat.com/browse/OCPBUGS-42812[*OCPBUGS-42812*])
+
+* Previously, if availability zones were not in a specific order in the `install-config.yaml` configuration file, the installation program would wrongly sort the zones before saving the control plane machine set manifests. When the program created the machines, additional control plane virtual machines were created to reconcile the machines into each zone. This caused a resource-constraint issue. With this release, the intallation program no longer sorts availability zones so that this issue no longer occurs. (link:https://issues.redhat.com/browse/OCPBUGS-42699[*OCPBUGS-42699*])
```

Comment:
```suggestion
* Previously, if availability zones were not in a specific order in the `install-config.yaml` configuration file, the installation program would wrongly sort the zones before saving the control plane machine set manifests. When the program created the machines, additional control plane virtual machines were created to reconcile the machines into each zone. This caused a resource-constraint issue. With this release, the installation program no longer sorts availability zones so that this issue no longer occurs. (link:https://issues.redhat.com/browse/OCPBUGS-42699[*OCPBUGS-42699*])
```